cg/meta/deliver.py

```diff
@@ -277,3 +280,28 @@ def create_delivery_dir_path(self, case_name: str = None, sample_name: str = Non
             delivery_path = delivery_path / sample_name
 
         return delivery_path
+
+    @staticmethod
+    def get_delivery_arguments(case_obj: models.Family) -> Set[str]:
+        """Translates the case data_delivery field to pipeline specific arguments."""
+        delivery_arguments: Set[str] = set()
+        requested_deliveries: List[str] = re.split("[-_]", case_obj.data_delivery)
```

Comment:
Should this not be a "get_case_deliveries" function somewhere on the case. I do not think any other module should know how to split the delivery string.

Reply:
I'm not sure I understand what you're suggesting. This is very tied to the actual delivery of the case so it's not really interesting for other modules, but could easily be moved to `models.py` if needed. Where would you prefer to have it?

Reply:
The format of the field data delivery is determined in the order form and then saved to the StatusDB field. I think only StatusDB should know of this format and any parsing should therefore be done by on the case object before showing it to a caller. That way if the format change only the parsing on the object need to change and all other calls remain the same. Basically, this code above would break if the format changed since it "knows" too much of the format (e.g. re.split("[-_]"), if it would just expect a list of strings instead (returned from the object), then it would not brake and become independent on code changing somewhere else.

Reply:
Makes sense. I will make it a case method then 👍